dashboard/app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -2,8 +2,6 @@
 
 <p>You can confirm your account email through the link below:</p>
 
-<p><%= link_to 'Confirm my account', confirmation_url(@resource, :confirmation_token => @token) %></p>
-
```

Comment:
This is attempting to do the minimal amount of work to remove `:confirmable` from the User model. This view, as well as other stuff, is being removed in #14512.